[2023-07-17T11:26:01.800+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main_dag.data_acquisition manual__2023-07-17T11:25:52.029317+00:00 [queued]>
[2023-07-17T11:26:01.810+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main_dag.data_acquisition manual__2023-07-17T11:25:52.029317+00:00 [queued]>
[2023-07-17T11:26:01.810+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-17T11:26:01.825+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): data_acquisition> on 2023-07-17 11:25:52.029317+00:00
[2023-07-17T11:26:01.829+0000] {standard_task_runner.py:57} INFO - Started process 1716 to run task
[2023-07-17T11:26:01.832+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'data_acquisition', 'manual__2023-07-17T11:25:52.029317+00:00', '--job-id', '654', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmpvkapxbpb']
[2023-07-17T11:26:01.836+0000] {standard_task_runner.py:85} INFO - Job 654: Subtask data_acquisition
[2023-07-17T11:26:01.876+0000] {task_command.py:410} INFO - Running <TaskInstance: main_dag.data_acquisition manual__2023-07-17T11:25:52.029317+00:00 [running]> on host a888360a1584
[2023-07-17T11:26:01.971+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main_dag' AIRFLOW_CTX_TASK_ID='data_acquisition' AIRFLOW_CTX_EXECUTION_DATE='2023-07-17T11:25:52.029317+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-17T11:25:52.029317+00:00'
[2023-07-17T11:27:38.702+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/wikipedia/wikipedia.py:389 GuessedAtParserWarning: No parser was explicitly specified, so I'm using the best available HTML parser for this system ("lxml"). This usually isn't a problem, but if you run this code on another system, or in a different virtual environment, it may use a different parser and behave differently.

The code that caused this warning is on line 389 of the file /home/***/.local/lib/python3.7/site-packages/wikipedia/wikipedia.py. To get rid of this warning, pass the additional argument 'features="lxml"' to the BeautifulSoup constructor.
[2023-07-17T11:27:38.714+0000] {logging_mixin.py:150} INFO - Could not load Casp
[2023-07-17T11:28:13.715+0000] {logging_mixin.py:150} INFO - Could not load Computer Vision
[2023-07-17T11:28:30.209+0000] {logging_mixin.py:150} INFO - Could not load Dbscan
[2023-07-17T11:28:39.220+0000] {logging_mixin.py:150} INFO - Could not load Data Mining
[2023-07-17T11:30:29.843+0000] {logging_mixin.py:150} INFO - Could not load Github
[2023-07-17T11:30:40.034+0000] {logging_mixin.py:150} INFO - Could not load Google
[2023-07-17T11:31:26.165+0000] {logging_mixin.py:150} INFO - Could not load Identity Matrix
[2023-07-17T11:32:53.111+0000] {logging_mixin.py:150} INFO - Could not load Machine Learning
[2023-07-17T11:33:32.434+0000] {logging_mixin.py:150} INFO - Could not load Multilayer Perceptron
[2023-07-17T11:33:38.578+0000] {logging_mixin.py:150} INFO - Could not load Np-Hard
[2023-07-17T11:34:35.106+0000] {logging_mixin.py:150} INFO - Could not load Pmc (Identifier)
[2023-07-17T11:34:42.678+0000] {logging_mixin.py:150} INFO - Could not load Petar Veličković
[2023-07-17T11:34:45.496+0000] {logging_mixin.py:150} INFO - Could not load Physics
[2023-07-17T11:34:49.196+0000] {logging_mixin.py:150} INFO - Could not load Pixel
[2023-07-17T11:35:32.086+0000] {logging_mixin.py:150} INFO - Could not load Relu
[2023-07-17T11:36:34.142+0000] {logging_mixin.py:150} INFO - Could not load Social Graph
[2023-07-17T11:37:14.517+0000] {logging_mixin.py:150} INFO - Could not load Tensorflow
[2023-07-17T11:37:16.347+0000] {logging_mixin.py:150} INFO - Could not load Thomas Kipf
[2023-07-17T11:37:47.149+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-07-17T11:37:47.163+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=data_acquisition, execution_date=20230717T112552, start_date=20230717T112601, end_date=20230717T113747
[2023-07-17T11:37:47.208+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-17T11:37:47.247+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
